f2/src/forms/MidFeedbackForm.js

```diff
@@ -1,60 +1,61 @@
 /** @jsx jsx */
 import React, { useState } from 'react'
+import PropTypes from 'prop-types'
 import { jsx } from '@emotion/core'
 import { H1, H2 } from '../components/header'
-import { useLingui } from '@lingui/react'
-import { Trans } from '@lingui/macro'
-import { Form } from 'react-final-form'
+import {
+  FeedbackButton,
+  MidFormFeedbackButton,
+} from '../components/formik/button'
 import { useLocation } from 'react-router-dom'
-import { Stack, Box } from '@chakra-ui/core'
-import { containsData } from '../utils/containsData'
-import { Button } from '../components/button'
-import { TextArea } from '../components/text-area'
+import { Trans } from '@lingui/macro'
+import { Form, Container, Row } from 'react-bootstrap'
+import { Formik, FieldArray, Field } from 'formik'
+import { CheckBox } from '../components/formik/checkbox'
 import { InfoCard } from '../components/container'
```

Comment:
Should use an Alert from f2\src\components\formik\alert\index.js. You may need to style a new component for this, let me know if you need help.